05_book_titles/book.rb

```diff
@@ -1,3 +1,28 @@
 class Book
-# write your code here
+	attr_accessor :title
+
+	public
+	def title=(name)
+		@title = t(name)
+	end
+
+	private
+	def t(title)
+		title = title.split(" ")
+
+		a_of_w = ['a', 'an', 'the', 'and', 'but', 'or', 'of', 'for', 'nor', 'etc.', 'in', 'on', 'at', 'to', 'from', 'by', 'over']
+		title.each do |word|		
+			word[0] = word[0].upcase unless a_of_w.include? word	
+		end
+
+		title[0].capitalize!
+		title = title.join(" ")	
+		title
+	end
 end
+
+my_book = Book.new
+
+puts my_book.title
+ my_book.title = "inferno"
+puts my_book.title
```

Comment:
Your function works well by itself, and if you add initalizer mentioned above, you can test you whole code like this:
```
my_book = Book.new('inferno')
puts my_book.title => "inferno"
puts my_book.t => "Inferno"
```